python/lsst/eotest/sensor/ptcTask.py

```diff
@@ -152,7 +152,8 @@ def run(self, sensor_id, infiles, mask_files, gains, binsize=1,
         exposure = []
         seqnums = []
         dayobs = []
-        file1s = sorted([item for item in infiles if item.find('flat1') != -1])
+        #file1s = sorted([item for item in infiles if item.find('flat1') != -1])
+        file1s = sorted([item for item in infiles if item.find('flat0') != -1])
```

Comment:
The code really isn't going to work in production with this change.   See https://github.com/lsst-camera-dh/eotest/pull/107/commits/76c4e11ff41aae736339fb9ab95a42e6980b942d

Reply:
Good catch.  I did not mean to leave in that change.

Reply:
I fixed it in the push just now.